resources/elasticbeanstalk_environments.go

```diff
@@ -226,9 +234,35 @@ func resolveElasticbeanstalkEnvironmentListeners(ctx context.Context, meta schem
 	}
 	return resource.Set(c.Name, listeners)
 }
-
+func resolveElasticbeanstalkEnvironmentListeners(ctx context.Context, meta schema.ClientMeta, resource *schema.Resource, c schema.Column) error {
+	p, ok := resource.Item.(types.EnvironmentDescription)
+	if !ok {
+		return fmt.Errorf("expected types.EnvironmentDescription but got %T", resource.Item)
+	}
+	region := meta.(*client.Client).Region
+	svc := meta.(*client.Client).Services().ElasticBeanstalk
+	tagsOutput, err := svc.ListTagsForResource(ctx, &elasticbeanstalk.ListTagsForResourceInput{
+		ResourceArn: p.EnvironmentArn,
+	}, func(o *elasticbeanstalk.Options) {
+		o.Region = region
+	})
```

Comment:
the client should be in the correct region, why are we also passing it as an option?